### PR TITLE
Bumped versions in pre-commit and npm configurations.

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

**Review by carltongibson — APPROVED** [review]
OK, super. 🏭

File: package.json [hunk 1]
```diff
@@ -9,10 +9,11 @@
     "npm": ">=1.3.0 <3.0.0"
   },
   "devDependencies": {
-    "eslint": "^7.32.0",
-    "grunt": "^1.0.1",
-    "grunt-cli": "^1.2.0",
-    "grunt-contrib-qunit": "^4.0.0",
-    "qunit": "^2.9.3"
+    "eslint": "^8.3.0",
+    "puppeteer": "^13.5.1",
```
Comment (thibaudcolas) [inline package.json:13]: Is it intentional for Puppeteer to be there? I can’t see it being used anywhere in the project.

Comment (felixxm) [reply package.json:13]: `puppeteer` is used by `grunt-contrib-qunit`. We wanted to pin a more recent version than required by `grunt-contrib-qunit`.

Comment (thibaudcolas) [reply package.json:13]: Thank you, that’s very interesting for potential automated accessibility tests as there are many more options with Puppeteer than Selenium.

Comment (felixxm) [reply package.json:13]: Also, the default version installed with `grunt-contrib-qunit` raises a warning:
```
npm WARN deprecated puppeteer@9.1.1: Version no longer supported. Upgrade to @latest
```

**Pull request merged**